archive-editorial.php

```diff
@@ -35,7 +35,7 @@
     the_post();
 ?>
 
-    <?php the_post_thumbnail('slide', array('data-id' => $post->ID, 'class' => 'archive-image')); ?>
+    <?php the_post_thumbnail( 'full', array('data-id' => $post->ID, 'class' => 'archive-image') ); ?>
```

Comment:
this means the image can be any amount of pixels big? 4k 5k etc? do the slides fit inside a columized space? if so we can have a image size that max width is the max of its container